vamb/__main__.py

```diff
@@ -1068,9 +1409,9 @@ def main():
         dest="model",
         metavar="",
         type=str,
-        choices=["vae", "aae", "vae-aae"],
+        choices=["vae", "aae", "vae-aae", "taxonomy_predictor", "vaevae", "reclustering"],
```

Comment:
It is possible that we might need to rethink the overall flow of `__main__.py`, if it needs to work with more kinds of models than just the VAE and the AAE. Perhaps it needs to be refactored to be more modular in terms of which models are being used. Let's discuss when we meet in person.